stable-profiles/0.9.52/disable-programs.local

```diff
@@ -0,0 +1,2 @@
+blacklist ${HOME}/.firejailed-tor-browser
+blacklist ${HOME}/.config/firejail
```

Comment:
Is `blacklist ${HOME}/.config/firejail` not blacklisted in 0.9.52 by default?

Reply:
I honestly don't know. I'm attempting to look up that information myself. However, if it is blacklisted by default then this statement is not needed. Apologies, I am still new to this 'firejail' program and your utility. With that said, I must defer to your experience and/or knowledge in this area.

Reply:
Also, I initially added that statement to that file because I didn't want other firejail'ed programs having access to that specific directory.

Reply:
Upon further investigation the `blacklist ${HOME}/.config/firejail` does not appear to be blacklisted by default in 0.9.52. I checked all disable-* files in `/etc/firejail/` did not find any matching statements. Also, when I initially began to work on this issue (while reading your instructions) I remember that I had to manually create this directory ` ${HOME}/.config/firejail/` because it did not exist. The only firejail related directory that existed by default was `${HOME}/.config/firetools/`. So maybe this is unique to 0.9.52? At any rate, I think it should be ok. What do you think?

Reply:
> I checked all disable-* files in /etc/firejail/ did not find any matching statements.

The blacklist is hardcoded, try `firejail --noprofile ls ~/.config/firejail -ld`.

Reply:
Ok here is my output from `firejail --noprofile ls ~/.config/firejail -ld`

```
$ firejail --noprofile ls ~/.config/firejail -ld
Parent pid 7982, child pid 7983
Child process initialized in 37.36 ms
dr-------- 2 root root 40 Oct  7 00:27 /home/user1/.config/firejail

Parent is shutting down, bye...
```

Reply:
It's blacklisted. All the `...${HOME}/.config/firejail` things can be removed.

Reply:
Ok I will remove them. Thank you!